R/utils.R

```diff
@@ -1,11 +1,20 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
 pv_query <- function(params, reformat = TRUE, ...){
+  if(length(params) > 1){
+    data <- lapply(as.list(params), pv_query, reformat = reformat)
+    if(reformat == TRUE){
+      return(do.call(rbind, data))
+    } else {
+      return(data)
```

Comment:
Alternately we could put this code in a function, `pv_query`, and create a separate function `pv_query_single` for a single query, which is not exported. Which ever way you prefer.